alert-common/src/main/java/com/synopsys/integration/alert/common/channel/issuetracker/message/IssueTrackerMessageParser.java

```diff
@@ -40,13 +42,20 @@
     private final int titleSizeLimit;
     private final int messageSizeLimit;
 
+    // TODO: What if description length differs from comment length? Should description length be accepted here?
```

Comment:
Let's create a Spike for this. This will become relevant when we begin implementing issue-trackers in the new channel framework.

Reply:
Upon looking at this further. It appears the message boards use the same limit for both description and comments. I think this should be addressed if it ever becomes an issue. I'm going to remove the TODO.